code/__DEFINES/radio.dm

```diff
@@ -21,27 +21,27 @@
 
 //Radio channels
 #define RADIO_CHANNEL_ALMAYER "Almayer"
-#define RADIO_CHANNEL_ALAMO "Alamo"	
-#define RADIO_CHANNEL_COMMAND "Command"
+#define RADIO_CHANNEL_ALAMO "Alamo"
+#define RADIO_CHANNEL_COMMAND ":v Command"
```

Comment:
These aren't used outside of radios and the define (as far as I can tell) *but* this should be done better.